crates/api_crud/src/community/read.rs

```diff
@@ -32,8 +28,10 @@ impl PerformCrud for GetCommunity {
       Some(id) => id,
       None => {
         let name = data.name.to_owned().unwrap_or_else(|| "main".to_string());
+        let community_actor_id = build_actor_id_from_shortname(EndpointType::Community, &name)?;
+
         blocking(context.pool(), move |conn| {
-          Community::read_from_name(conn, &name)
+          Community::read_from_apub_id(conn, &community_actor_id)
```

Comment:
Now instead of receiving a local only name, we receive a `dessalines` or `dessalines@xyz.tld`, and it builds either the local or remote actor_id url from it.

I then filter on actor id instead of name.